client/src/api/tutorial/steps/gamePlay.ts

```diff
@@ -2,25 +2,47 @@ import { Step, LEFT } from '@port-of-mars/client/types/tutorial';
 import { CURATOR } from '@port-of-mars/shared/types';
 
 const steps: Array<Step> = [
+  {
+    target: '.tour-game-status',
```

Comment:
the tutorial isn't starting - I think you forgot to set this class attribute on an html element

Reply:
haha 😅 thanks!